src/olympia/migrations/1126-db-schema-fixes.sql

```diff
@@ -0,0 +1,137 @@
+-- Model missing for table: compat_override_range
+DROP TABLE IF EXISTS  `compat_override_range`;
+-- Model missing for table: compat_override
+DROP TABLE IF EXISTS `compat_override`;
+-- Model missing for table: email_preview
+DROP TABLE IF EXISTS  `email_preview`;
+-- Model missing for table: incompatible_versions
+DROP TABLE IF EXISTS  `incompatible_versions`;
+-- Model missing for table: product_details_productdetailsfile
+DROP TABLE IF EXISTS  `product_details_productdetailsfile`;
+-- Model missing for table: webext_permission_descriptions
+DROP TABLE  IF EXISTS `webext_permission_descriptions`;
+-- Model missing for table: zadmin_siteevent
+DROP TABLE  IF EXISTS `zadmin_siteevent`;
+
+ALTER TABLE `abuse_reports`
+    MODIFY `addon_id` int(10) unsigned DEFAULT NULL,
+    DROP FOREIGN KEY `reporter_id_refs_id_12d88e23`,
+    DROP FOREIGN KEY `user_id_refs_id_12d88e23`,
+    DROP FOREIGN KEY `addon_id_refs_id_2b6ff2a7`,
+    DROP KEY `created_idx`;
+ALTER TABLE `abuse_reports`
+    ADD CONSTRAINT `abuse_reports_addon_id_f15faa13_fk_addons_id` FOREIGN KEY (`addon_id`) REFERENCES `addons` (`id`),
+    ADD CONSTRAINT `abuse_reports_reporter_id_e5b6b72a_fk_users_id` FOREIGN KEY (`reporter_id`) REFERENCES `users` (`id`),
+    ADD CONSTRAINT `abuse_reports_user_id_67401662_fk_users_id` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`);
+
+ALTER TABLE `addons`
+    MODIFY `defaultlocale` varchar(10) NOT NULL,
+    MODIFY `icontype` varchar(25) NOT NULL,
+    MODIFY `weeklydownloads` int(10) unsigned NOT NULL,
+    MODIFY `hotness` double NOT NULL,
+    MODIFY `experimental` tinyint(1) NOT NULL,
+    DROP FOREIGN KEY `addons_ibfk_2`,  /* name */
+    DROP FOREIGN KEY `addons_ibfk_3`,  /* homepage */
+    DROP FOREIGN KEY `addons_ibfk_4`,  /* description */
+    DROP FOREIGN KEY `addons_ibfk_5`,  /* summary */
+    DROP FOREIGN KEY `addons_ibfk_6`,  /* developercomments */
+    DROP FOREIGN KEY `addons_ibfk_7`,  /* eula */
+    DROP FOREIGN KEY `addons_ibfk_8`,  /* privacypolicy */
+    DROP FOREIGN KEY `addons_ibfk_9`,  /* supporturl */
+    DROP FOREIGN KEY `addons_ibfk_10`,  /* supportemail */
+    DROP FOREIGN KEY `addons_ibfk_14`,  /* current_version */
+    DROP KEY `created_idx`,
+    DROP KEY `modified_idx`;
```

Comment:
We probably need these 2, don't drop them, add them to indexes on the model

Reply:
I wasn't sure about them - we only had them for a few tables and they seemed more like historic cruft rather that something useful.

Reply:
But sure, can do.